client/allocrunner/csi_hook.go

```diff
@@ -158,6 +161,7 @@ func (c *csiHook) claimVolumesFromAlloc() (map[string]*volumeAndRequest, error)
 			Claim:        claimType,
 		}
 		req.Region = c.alloc.Job.Region
+		req.AuthToken = c.ar.clientConfig.Node.SecretID
```

Comment:
It wasn't part of this bug, but are we missing the namespace here?

Reply:
The `Node.SecretID` bypasses all the namespace specific checks for auth, which is why I omitted it originally. Looking though, we do use the request namespace to find the volume later. So the answer is yes, we need it.